src/index.js

```diff
@@ -0,0 +1,35 @@
+import './style.css';
+
+const arrayOfObjects = [
+  {
+    description: 'Wash dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Clean the entire building',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Doing laundry',
+    completed: false,
+    index: 2,
+  },
+];
+
```

Comment:
- [ ] Please make sure to iterate over the tasks array and populate the HTML list item element for each task. Currently, I can't see these tasks on the UI.

![image](https://user-images.githubusercontent.com/83803414/228531547-d47cd0b9-bd02-4b29-8df5-e00e492b7149.png)